### PR TITLE
mgr/progress: Global Recovery Event in ceph -s

### PR DESCRIPTION
**Pull request by kamoltat**
Only show Global Progress event in ceph -s
Right now if you want to see all the sub events
You have to do ceph progress.

Todo: add the command ceph progress detail and json version
update qa test case for test_progress.py

Signed-off-by: Kamoltat (Junior) Sirivadhna <ksirivad@redhat.com>

## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test docs`
- `jenkins render docs`

</details>

### CODE REVIEW

File: src/pybind/mgr/progress/module.py [hunk 1]
```diff
@@ -24,19 +23,20 @@ class Event(object):
     objects (osds, pools) this relates to.
     """
 
-    def __init__(self, message, refs, started_at=None):
+    def __init__(self, message, refs, started_at=None, add_to_ceph_s=False):
```
Comment (jdurgin) [inline src/pybind/mgr/progress/module.py]: let's default add_to_ceph_s to true for now, we can start excluding events if there are too many later

**Comment by sebastian-philipp** [comment]
@jdurgin I guess this PR is abandoned?

**Comment by liewegas** [comment]
@jdurgin want to adopt this?

**Comment by jdurgin** [comment]
@liewegas it's on my list

**Comment by sebastian-philipp** [comment]
Note that `qa/tasks/mgr/test_progress.py` was moved to `src/pybind/mgr/progress`

**Comment by kamoltat** [comment]
Closing this PR and Move to #37327

**Pull request closed without merging**